scripts/build/bundler.js

```diff
@@ -37,6 +39,33 @@ const EXTERNALS = [
   "graceful-fs",
 ];
 
+const entries = [
```

Comment:
What is this change doing in this PR?

Reply:
The entries of `outdent` need point to commonjs build,  and I tried fix in 4 bundles, still not enough, seems it's
 imported everywhere, so I move them to commom alias.